src/JsonStringPlaceholderTemplate.php

```diff
@@ -101,16 +93,17 @@ public function __construct($template, $tokenStart, $tokenEnd, $tokenDefault, $j
      */
     public function render($context = null)
     {
-        if ($context) {
-            foreach ($this->jsonStringFields as $jsonStringField) {
-                if (!$this->_containerHas($context, $jsonStringField)) {
-                    continue;
-                }
-
-                $this->_containerSet($context, $jsonStringField, json_encode($this->_containerGet($context, $jsonStringField)));
-            }
+        if (!$context) {
+            return parent::render($context);
+        }
+
+        $transformedContext = [];
+        $jsonKeys           = $this->_normalizeArray($this->jsonStringFields);
+
+        foreach ($context as $key => $value) {
```

Comment:
[Container is not iterable](https://github.com/Dhii/output-renderer-interface/blob/develop/src/TemplateInterface.php#L29).